README.md

```diff
@@ -28,6 +30,16 @@ const pluginSecurity = require('eslint-plugin-security');
 module.exports = [pluginSecurity.configs.recommended];
 ```
 
+### eslintrc config(legacy)
+
+Add the following to your `.eslintrc` file:
+
+```js
+"extends": [
+  "plugin:security/recommended-legacy"
+]
+```
```

Comment:
Let's make this a full file:

```suggestion
```js
module.exports = {
  "extends": [
    "plugin:security/recommended-legacy"
  ]
}
```
```